CommandlineTool/main.swift

```diff
@@ -61,13 +61,24 @@ struct xcresultparser: ParsableCommand {
             } else {
                 try outputJUnitXML(for: xcresult)
             }
+        } else if format == .cobertura {
+            if coverage != 1 { throw ParseError.argumentError }
```

Comment:
I was thinking about that, I decided to just add a simple check as a first pass because I didn't want to look into mutating the flags in this scenario, but I agree setting coverage automatically would be more user-friendly. I will look into it.